src/index.css

```diff
@@ -225,3 +224,24 @@ body {
   justify-content: center;
   align-items: center;
 }
+
```

Comment:
- [ ] It would be better to properly align the contents in the Navbar to the books, for example; the title `BookstoreCMS` and the avatar must align with the book container soo that we can achieve the desired design

Reply:
Sure, I missed that part. It's aligned correctly now.